### ISSUE TITLE
Pagan Subjugation seems to be using defensive holy war call-to-arms mechanic against same-religion foes.

### ISSUE THREAD

**Issue by zijistark** [zijistark/EMF#604]
![image](https://user-images.githubusercontent.com/6268890/52521964-2c05a600-2c34-11e9-80af-3c1b412681e4.png)

Notice all of the "glowing" red hostility/war highlighting around the different independent realms in Scandinavia.

**Comment by zijistark** [comment]
This behavior was intended for different-religion pagans, BTW. I'm writing a second CB that's a diff-religion version which still uses the coreligionist call-to-arms mechanic.

**Comment by zijistark** [comment]
I wrote a second CB, `emf_pagan_subjugation_religious` which is for Subjugation wars against different religions (while the regular one is now only for same-religion targets). The religious variant still uses the coreligionist defensive call to arms mechanic. This is motivated by:

- Pagans have always been very blobby, and I'm trying to make that less of a thing
- The Pagan Subjugation CB converts the target to the attacker religion upon success, if they are the reformed version of their pagan religion. So, it's a bit of a religious CB in its own right.

Additionally, I added a prestige cost to both CBs that scales with the number of provinces in the [independent] realm of the [independent] defender. This makes sense since the entire defender realm is consumed upon victory. The prestige cost is essentially 25 prestige per province, up to a cap of 500 prestige cost for a defender with more than 16 realm provinces.  The AI pays slightly less vs. large realms, capping out at 350 prestige cost. [I don't want to discourage the AI from attacking such realms, since it's not nearly as good as a player at gaming prestige gain & savings, especially with tribal retinues costing prestige now.]

I also noticed that the CB was lacking any `is_valid` clause, which is very very bad and could lead to totally ridiculous wars that literally cannot end. Fixed.
